README.md

```diff
@@ -133,6 +133,30 @@ If you want to ignore the same files and folders contained in the `.gitignore` f
 eslint --ignore-path .gitignore **/*.js
 ```
 
+### Per-project rules
+
+We've chosen a sensible set of rules and plugins that helps us catch the most common errors while writing JavaScript. Using a unified linting configuration across Springer Nature makes it easier to share code and resources between different teams and projects.
+
+In certain situations it may be necessary to specify per-project rules. A typical use case is when migrating an old project using a different linter to this eslint config, where it may not be possible to fix all the issues raised by eslint in one go and we opt instead for an iterative approach.
+
+If you need to add per-project rules, consider changing them from `error` to `warn` instead of disabling them completely. This will allow any tests to pass, but will help you remember that a rule has been overwritten:
+
+```js
+  "rules": {
+    "block-scoped-var": "warn",
+  }
+```
+
+If the problem is in one specific line of code, consider wrapping the offending code in `eslint-disable` directives instead of disabling a rule for your whole project, for example:
```

Comment:
👍 but would `// eslint-disable-line ...` be a better candidate for this example? It’s concise and it isn’t vulnerable to forgetting to turn the rule back on.

Reply:
YES! The only reason I didn't include that instead is because I forgot that it existed. 😂 

I've made that change now, thanks for spotting this!

Reply:
Hey Jose, thanks, but the comment and the code example don't seem to fit together now, sorry 😬 

I'd either:
- change the example itself to use `// eslint-disable-line` (note this must be at the end of the line of code, or it would have to be `eslint-disable-next-line`)
- just bung all the different methods into the example

Reply:
Absolutely right! I've just created #33 to address this.

Thanks for catching this!